README.md

```diff
@@ -1 +1,182 @@
+
+<details>
+<summary>Interaction and Dynamic HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript">Javascript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Some useful features
+
+- **[Using Grid and Flexbox for the layout]**
+- **[HTML]**
+- **[CSS]**
+- **[JAVASCRIPT]**
+
+
+<!-- LIVE DEMO -->
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Visual Studio Code must be installed on your computer | Download Visual Studio Code [VSCode](https://code.visualstudio.com/)
+- Git must be installed on your computer | Download [Git](https://git-scm.com/downloads)
+
+### Setup
+- Clone the repository
+- Open the project folder in Visual Studio Code
+
+### Install
+
+Install this project using:
+
+- cd project/directory
+- Git clone [Module 1 Capstone Project](https://github.com/jecode93/first-module-capstone-project.git)
+
+### Usage
+
+To run the project, execute the following command:
+
+- Launch index.html file in your browser
+
+
+### Run tests
+
+To run tests, run the following command:
+
+- Open the index.html file
+- Modify the name in the h1 tag or the description
+- Refresh you browser to see the changes
+
+### Deployment
+
+You can deploy this project using:
+
+- [Netlify](https://www.netlify.com/)
+- [GitHub Page](https://pages.github.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Project documentation and short descriptive Video
+
+- Click on the link for [Short video](https://www.loom.com/share/023a8b7c06d44473acfe5079efe04975?sid=bbc06854-132d-4726-b26f-2a178992ad75)
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link will be available soon]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## Authors
+
+
+👤 **Jean Emmanuel Cadet**
+
+- GitHub: [@jecode93](https://github.com/jecode93)
+- Twitter: [@jecode93](https://twitter.com/jecode93)
+- LinkedIn: [jean-emmanuel-cadet](https://www.linkedin.com/in/jean-emmanuel-cadet/)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Mobile version]**
+- [ ] **[Desktop version]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to the Microverse team for the great curriculum.
+- Thanks to the Code Reviewer(s) for the insightful feedback.
+- A great thanks to My coding partner(s), morning session team, and standup team for their contributions.
+- [Cindy Shin](https://www.behance.net/adagio07) the author of the original design [www.behance.net](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015/)
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] @jecode93, Fantastic working on the changes requested.  I think the main reason to why your Link to the mit is raising a `404` page error Is because of the pathname. The pathname should look like  `[MIT](/LICENSE)`. Kindly remove the last extension `.md` as it is not part of its name.

Reply:
Hi @frank1738 ,

Done!